### DISCUSSION TITLE
skoala/ms-gateway/gateway/create-gateway/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1531] [Announcements]
# skoala/ms-gateway/gateway/create-gateway/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/skoala/ms-gateway/gateway/create-gateway/

**Comment by liyuerich** [comment]
dce0 0.7.0 使用云原生网关，不是文档上的微服务网关

**Reply by Michelle951** [reply]
fixed by pull request: https://github.com/DaoCloud/DaoCloud-docs/pull/1893
